packages/hash/frontend/src/pages/[account-slug]/entities/[entity-id].page/entity-editor/properties-section/get-empty-property-count.ts

```diff
@@ -0,0 +1,15 @@
+const isValueEmpty = (val: any) => {
+  if (val === "" || val === null || val === undefined) {
+    return true;
+  }
+
+  if (Array.isArray(val) && !val.length) {
+    return true;
+  }
+
+  return false;
```

Comment:
I think this will need updating for nested objects, e.g.
```
{ address: {} }
```
or
```
{ address: { street: undefined, city: undefined } }
```
`address` is not counted as an empty property at the moment